vhdbuilder/packer/install-dependencies.sh

```diff
@@ -236,6 +236,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.2.5
```

Comment:
@vakalapa  Do we still need 1.1.8 and 1.1.7 ? If not can we remove it?

Reply:
Cleaned up some stale versions.